linode/manager
# M3-5592: Update PaymentMethodRow story to mdx

## Description
Updates the `PaymentMethodRow` story from `tsx` to `mdx`

## How to test
- `yarn storybook`
- Go to http://localhost:6006/?path=/docs/components-paymentmethodrow and check all the stories
